### DISCUSSION TITLE
Confuser password leaking, maybe

### DISCUSSION THREAD

**Discussion by alex6dj** [mkaring/ConfuserEx#461] [General]
Hi, I already notice that after use confuser cli with rename mode reversible and password set, a file called password with the clear password used is left in the output project folder. I don´t know if this was the intended behaviour but is a dangerous one wich need at least be documented.

**Comment by mkaring** [comment]
That seems like a bug. I'll have to look into it. Writing the password is intended, but only in case it's auto generated. It shouldn't write the password if it's set to a fixed value.

**Reply by alex6dj** [reply]
Let me know if more information is needed.
